modules/ROOT/pages/managing-application-data-with-object-stores.adoc

```diff
@@ -49,19 +50,28 @@ To make properties secure so that they're not visible using the Runtime Manager
 
 == Semantics and Storage Limits
 
-The Runtime Manager object store provides read-after-write consistency and the last write wins in terms of updates.
+Object Store v1 provides read-after-write consistency and the last write wins for updates.
 
-Additionally, Runtime Manager  object store places these limits on your data:
+Object Store v1 has these limits:
 
-* A maximum of 100,000 keys per application
-* A 768 byte key size
-* A 1 MB value size
+* Maximum of 100,000 keys per application
+* 768 byte key size
+* 1 MB value size
 * 1 GB of data per application
 
+Object Store v2 has these limits:
+
+* Works with Mule 3.8.5 and later, and Mule 4
+* Unlimited number of keys per application
+* Values can be up to 10 MB
+* Trial level 10 TPS, premium level 100 TPS
+* Keys persist for up to 30 days
+* Available in 10 regions around the world 
```

Comment:
Available in all commercially available Cloudhub regions.

Reply:
Changed to:
Available in xref:object-store::osv2-faq.adoc[all commercially available CloudHub regions].

The FAQ's first section lists what regions are available.